### PR TITLE
Task9view by category

### PR DESCRIPTION
**Pull request by FameTii**
has dropdown menu to filter by categories

### CODE REVIEW

File: src/Api.js [hunk 1]
```diff
@@ -33,10 +33,24 @@ const getComments = (reviewId) => {
...
+    .patch(
+      `https://fame-boardgame-review-website.onrender.com/api/reviews/${reviewId}`,
+      { inc_votes: value }
+    )
+    .then(({ data }) => {
+      return data.inc_votes;
+    });
+};
+
+const getCategory = (category) => {
```
Comment (JVal-NC) [inline src/Api.js]: Could getReviews be modified to allow for an optional category argument?

File: src/SortByCategory.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  let { category } = useParams();
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCategory(category).then((data) => {
+      setReviews(data);
+      setIsLoading(false);
+    });
+  }, []);
```
Comment (JVal-NC) [inline src/SortByCategory.jsx]: Think we need something inside this dependancy array. What happens if we try to change the category drop down more than once?

File: src/CategoryDropDown.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    redirectURL();
+  }, [category]);
+
+  const redirectURL = () => {
+    console.log(category);
+    if (category === "") {
+      return navigate(`/`);
```
Comment (JVal-NC) [inline src/CategoryDropDown.jsx]: I think this (line 15) might be navigating back to the home page when refreshing the page which is not desirable.

We want the URL to take another user to the same place within the app.

**Review by JVal-NC — COMMENTED** [review]
Good work, a couple comments to consider.

**Review by pippaamy — APPROVED** [review]
Great changes!

**Pull request merged**